### PR TITLE
feat:add discord icon to header

### PR DESCRIPTION
**Pull request by clavy123**
issue #1443

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
just change the position @clavy123 

image

File: src/common/header/header.css [hunk 1]
```diff
@@ -267,6 +267,10 @@
   fill: #ffffff;
 }
 
+.header-links > li > .app-header-btn:hover .discord-icon {
+  fill: #ffffff;
```
Comment (priyankarpal) [inline src/common/header/header.css:271]: it should be a discord color `#7289da`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by clavy123** [comment]
@priyankarpal  changes done!!!

**Comment by priyankarpal** [comment]
> @priyankarpal changes done!!!

yes let me check

**Comment by priyankarpal** [comment]
hello @clavy123 there is a conflict could you please fix it?

**Pull request merged**